unix/runner

```diff
@@ -32,10 +38,14 @@ def _size(key, mounts):
 
 def storage_allocated_gb(facts):
     """Return total storage allocation in GB"""
+    if 'ansible_mounts' not in facts:
+        return 0
     return _size('size_total', facts['ansible_mounts']) / 1024**3
 
 def storage_used_gb(facts):
     """Return used storage in GB"""
+    if 'ansible_mounts' not in facts:
+        return 0
```

Comment:
I wonder if we can _also_ log an error here?

I would like to know which servers didn't have `ansible_mounts` available for further remediation.

Reply:
@dcolebatch please see https://github.com/tidalmigrations/machine_stats/pull/11/commits/4e69522cf7d043ec4ae22870642fe42e00ee5978